kernel/arch/dreamcast/fs/fs_iso9660.c

```diff
@@ -578,10 +583,13 @@ static struct {
     uint32      size;       /* Length of file in bytes */
     dirent_t    dirent;     /* A static dirent to pass back to clients */
     int     broken;     /* >0 if the CD has been swapped out since open */
+    size_t     stream_part;     /* Stream DMA part of 32 bytes */
+    uint8_t stream_data[32] __attribute__((aligned(32)));
```

Comment:
```suggestion
    alignas(32) uint8_t stream_data[32];
```
Since KOS got bumped up to `C17`, we now get alignment as part of the language...

Just be sure to include `<stdalign.h>`.